lib/full-site-editing/template-parts.php

```diff
@@ -266,3 +266,83 @@ function gutenberg_block_header_area() {
 function gutenberg_block_footer_area() {
 	gutenberg_block_template_part( 'footer' );
 }
+
+register_block_pattern(
+	'template-part/header-test',
+	array(
```

Comment:
Don't forget to remove these before merging. We can handle this in the related issue https://github.com/WordPress/gutenberg/issues/31231